source/s3_meta_request.c

```diff
@@ -310,6 +519,22 @@ PyObject *aws_py_s3_client_make_meta_request(PyObject *self, PyObject *args) {
     meta_request->self_py = py_s3_request;
     Py_INCREF(meta_request->self_py);
 
+    if (recv_filepath) {
+        meta_request->recv_file = fopen(recv_filepath, "wb+");
+    }
+    if (send_filepath) {
+        meta_request->send_file = fopen(send_filepath, "rb+");
```

Comment:
likewise, not checking results of fopen, but you probably don't need this code anyway...